packages/ui/src/stories/Button/Button.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import { storiesOf } from "@storybook/react";
+import { Button } from "./Button";
+
+storiesOf("Button", module)
+  .add("primary", () => <Button>Call to Action</Button>)
+  .add("primary disabled", () => <Button disabled>Call to Action</Button>)
+  .add("secondary", () => <Button type="secondary">Call to Action</Button>)
+  .add("secondary disabled", () => (
+    <Button disabled type="secondary">
+      Call to Action
+    </Button>
+  ))
+  .add("onClick handler", () => (
+    // eslint-disable-next-line no-alert
```

Comment:
This is okay here since this is simply a story, but keep in mind that in actual code we would want to avoid disabling eslint 99.9 percent of the time :)

Reply:
You are right. I deleted already. I didn't think about it because you used it in the web-components repo. I left it there because I didn't do enough research on what that could mean, it was incorrect of me to do so.
I am still getting the sense of eslint and how to deal with it. I was reading yesterday until 2am.  But of course I shouldn't have left code I don't recognize.

Reply:
Think of eslint as a set of rules people set up in their projects that warns you about best practises.